packages/expo/android/src/main/java/expo/modules/color/ExpoMaterialColors.kt

```diff
@@ -0,0 +1,132 @@
+package expo.modules.color
+
+import android.content.Context
+import android.content.res.Configuration
+import android.graphics.Color
+import expo.modules.kotlin.exception.Exceptions
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+import androidx.appcompat.view.ContextThemeWrapper
+import com.google.android.material.color.MaterialColors
+import com.google.android.material.color.DynamicColors
+import com.google.android.material.color.DynamicColorsOptions
+
+
+class ExpoMaterialColors : Module() {
+  private val context: Context
+    get() = appContext.reactContext ?: throw Exceptions.ReactContextLost()
+
+  override fun definition() = ModuleDefinition {
+    Name("ExpoMaterialColors")
+
+    Function("Material3Color") { name: String, scheme: String ->
+      materialColor(name, scheme)
+    }
+
+    Function("Material3DynamicColor") { name: String, scheme: String ->
+      dynamicColor(name, scheme)
+    }
+  }
+
+  private fun materialColor(name: String, scheme: String): String? {
+    var scheme = scheme
+    if (scheme != "dark" && scheme != "light") {
+      scheme = getSystemScheme()
+    }
```

Comment:
```suggestion
    val scheme = if (scheme != "dark" && scheme != "light") {
      getSystemScheme()
    } else {
      scheme
    }
```
Or convert it to a nullable enum.